packages/contracts/test/ERC20KPITokenInitializeTest.sol

```diff
@@ -28,7 +29,7 @@ contract ERC20KPITokenInitializeTest is BaseTestSetup {
                 kpiTokenTemplateVersion: 1,
                 description: "a",
                 expiration: block.timestamp + 60,
-                kpiTokenData: abi.encode(collaterals, "Token", "TKN", 100 ether),
+                kpiTokenData: abi.encode(rewards, "Token", "TKN", 100 ether),
```

Comment:
Aren't we missing the `_jitFunding` here? In the `abi.encode()`.

Reply:
Should be addressed in b899106e6cd269c8a3110a534a7064440c865e89